### ISSUE TITLE
Documentation website colors are eye-straining on Firefox on a Mac

### ISSUE THREAD

**Issue by richardxia** [styled-components/polished#573]
## Summary

**tl;dr: Firefox on a Mac has a bug that causes the green background on the polished documentation site to be blindingly bright. Would you be open to changing the color?**

I think this bug may only affect computers with displays that support a wide range of colors, which notably includes almost all Macs. On a 2019 16" MacBook Pro, which has a builtin display that supports the [P3 color space](https://en.wikipedia.org/wiki/DCI-P3), Firefox will incorrectly render the background color of the [documentation website](https://polished.js.org/docs/) in a relatively bright and saturated green, as opposed to Chrome and Safari, which will correctly render it in a duller green.

 
## Basic Example

Here are some comparison screenshots:

### Firefox

<img width="1047" alt="Screen Shot 2021-04-05 at 1 18 14 PM" src="https://user-images.githubusercontent.com/1002748/113624121-158ce680-9614-11eb-886d-89345a924f3b.png">

### Chrome

<img width="1109" alt="Screen Shot 2021-04-05 at 1 18 22 PM" src="https://user-images.githubusercontent.com/1002748/113624135-1aea3100-9614-11eb-950d-b51477d021ed.png">

Note that the screenshots don't quite do it justice, since viewing it a web browser that covers more of the screen makes the background color feel even brighter, and it tires my eyes more the longer I view it.

## Reasoning

It seems that this may be an [issue in Firefox](https://apple.stackexchange.com/questions/327808/why-do-colors-appear-washed-out-on-macos-chrome-and-safari-versus-firefox), where Firefox is not properly translating the CSS colors from the web's native color space, the [sRGB color space](https://en.wikipedia.org/wiki/SRGB), to the display's native color space, in my case P3.

I used the macOS Digital Color Meter application to measure the actual rendered colors expressed in the sRGB color space, and I can confirm that Firefox is doing something weird:

### Firefox
<img width="897" alt="Screen Shot 2021-04-05 at 1 23 21 PM" src="https://user-images.githubusercontent.com/1002748/113624516-9c41c380-9614-11eb-9733-9cda84b00acd.png">

### Chrome
<img width="800" alt="Screen Shot 2021-04-05 at 1 22 36 PM" src="https://user-images.githubusercontent.com/1002748/113624522-9e0b8700-9614-11eb-92ad-073d2b5f533e.png">

As you can see, the actual rendered colors on Chrome almost exactly match the hex colors specified in the CSS (`#65daa2` == `rgb(101, 218, 162)`), but Firefox ends up with an extremely low value for the red channel. This is likely due to the fact that the P3 color space covers a lot more in the green area of the visible spectrum of colors, so when Firefox doesn't properly translate colors, the greens end up being distorted the most. See the following diagram of the color spaces overlaid on top of each other:
![2d-srgb-applep3](https://user-images.githubusercontent.com/1002748/113626523-1ffcaf80-9617-11eb-870b-c895c7a7323d.jpg)

(Image credit: http://www.astramael.com/)

## Suggested fixes

If you could pick a less saturated green for the background, I think that would help make the Firefox rendering of the page less saturated, even if the color is still a bit wrong in Firefox.

Another solution would be to pick a primary brand color with a hue that is further away from pure green, since the distortions would be smaller anywhere else in the color space.

I realize that this is strange request, and I understand if this isn't something you want to change, especially because this ultimately is a Firefox bug that only affects certain displays. I certainly work around this by just viewing the docs in a different browser than Firefox.
